parlai/core/torch_agent.py

```diff
@@ -253,8 +276,9 @@ def vectorize(self, obs, add_start=True, add_end=True, truncate=None,
         :param split_lines: If set, returns list of vectors instead of a single
                             vector for input text, one for each substring after
                             splitting on newlines.
+        :param nocache:     default False, recalculates vecs even if present
         """
-        if 'text_vec' in obs:
+        if 'text_vec' in obs and not nocache:
```

Comment:
Interesting. Why?

Reply:
I manually modified the eval_labels field in the PerplexityEvaluator and wanted to re-vectorize. However I realized I could just call the _vectorize_text function... I'll revert this.